jvb/src/main/kotlin/org/jitsi/videobridge/cc/allocation/SingleSourceAllocation.kt

```diff
@@ -255,13 +264,20 @@ private fun selectLayersForScreensharing(
     return Layers(selectedLayers, selectedLayers.size - 1, oversendIdx)
 }
 
+/** Return the index of the first item in the list which satisfies a predicate, or -1 if none do. */
 private fun <T> List<T>.firstIndexWhich(predicate: (T) -> Boolean): Int {
     forEachIndexed { index, item ->
         if (predicate(item)) return index
     }
     return -1
 }
 
+/**
+ * Selects from a list of layers the ones which should be considered when allocating bandwidth, as well as the
+ * "preferred" and "oversend" layers. Logic specific to a camera stream: once the "preferred" height is reached we
+ * require a high frame rate, we preconfigured values for the "preferred" height and frame rate, and we do not allow
```

Comment:
I don't understand the "we preconfigured" phrase - should this be in the present tense?

Reply:
I meant to write  "with preconfigured values". Is this clear?